migrations/058-arb-all-current-votes-b.sql

```diff
@@ -0,0 +1,60 @@
+DROP FUNCTION IF EXISTS api.all_current_votes(CHAR);
+
+--testing to replace 38
+CREATE OR REPLACE FUNCTION api.all_current_votes(arg_address CHAR)
+RETURNS TABLE (
+	poll_id integer,
+	option_id_raw character,
+	option_id integer,
+	block_timestamp timestamp with time zone,
```

Comment:
what's the difference between this and the other allCurrentVotes query?

Reply:
each migration replaces the previous functions, the newest one is `060-arb-all-current-votes-c.sql`, so the ones before it are obsolete.

Reply:
can't we just delete them? since when we merge the PR it will only use the latest one